File: Python/linear_search.py
def linear_search(arr, target):
    for i in range(len(arr)):
        if arr[i] == target:
            return i
    return -1

my_list = [1, 5, 7, 9, 11, 13]
target_element = 9

result = linear_search(my_list, target_element)

if result != -1:
    print(f"{target_element} found at index {result}.")
else:
    print(f"{target_element} not found in the list.")

File: Java/LinearSearch.java
import java.util.*;
public class LinearSearch {
    public static void main(String args[]) {
        // taking user input
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter number of elements:");
        int N = sc.nextInt();
        System.out.println("Enter elements:");
        
        // storing elements in an array
        int[] arr = new int[N];
        for(int i=0;i<N;i++){
            arr[i] = sc.nextInt();
        }
        
        // asking user for target element
        System.out.println("Enter target element:");
        int target = sc.nextInt();
        
        // using linear search to find element
        int index = Search(N, arr, target);
        
        if(index == -1)
            System.out.println("Sorry, element not found");
        else
            System.out.println("The element is present at index "+index);
    }
    
    public static int Search(int N, int[] arr, int target){
        
        // using for loop to traverse the array
        for(int i=0; i<N; i++){
            // if target is found return
            if(arr[i] == target) return i+1;
        }
        // if the target is not present return 
        return -1;
    }
}
